src/containers/AppWithClusters/AppWithClusters.tsx

```diff
@@ -31,7 +31,7 @@ const defaultUserSettings = settings;
 
 defaultUserSettings[2].sections[0].settings.push({
     title: i18n('settings.useClusterBalancerAsBackend.title'),
-    helpPopoverContent: i18n('settings.useClusterBalancerAsBackend.popover'),
+    description: i18n('settings.useClusterBalancerAsBackend.popover'),
```

Comment:
What do you think about move description to the right section? It's more place there.
<img width="597" alt="Screenshot 2024-07-03 at 08 52 11" src="https://github.com/ydb-platform/ydb-embedded-ui/assets/42094364/f466d33a-885b-4d29-8f42-32b3dae5fb41">

Reply:
It looks good. However, I'd prefer to do it further, since it's not just an option in uikit component and requires some coding, and now we have quite enough space to display everything without scroll. Added issue to discuss it later: https://github.com/ydb-platform/ydb-embedded-ui/issues/974